libs/anglify/src/modules/toolbar/toolbar.component.scss

```diff
@@ -0,0 +1,77 @@
+@use 'variables' as *;
+@use 'sass:map';
+@use '../../styles/settings/typography.scss' as *;
+
+:host {
+  display: flex;
+  justify-content: space-between;
+
+  .toolbar-container {
+    height: $toolbar-regular-height;
+    background-color: $toolbar-background;
+    color: $toolbar-text-color;
+    width: $toolbar-width;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    .left {
+      display: flex;
+      height: 100%;
+      align-items: center;
+      .navigation {
+        margin-left: $toolbar-navigation-margin-left;
+      }
+      .navigation ::ng-deep * {
+        color: $toolbar-text-color;
+      }
+      .title {
+        margin-left: $toolbar-title-margin-left;
+      }
+    }
+    .right {
+      display: flex;
+      height: 100%;
+      margin-right: $toolbar-right-actions-margin-right;
+      align-items: center;
+      .actions {
+        margin-left: $toolbar-actions-margin-left;
+      }
+      .actions ::ng-deep * {
+        color: $toolbar-text-color;
+      }
+    }
+  }
+  .prominent {
+    height: $toolbar-prominent-height;
+    padding-top: $toolbar-prominent-padding-top;
+    .left {
+      align-items: flex-start;
+      .title {
+        align-self: end;
+        margin-bottom: $toolbar-prominent-title-margin-bottom;
+      }
+    }
+    .right {
+      align-items: flex-start;
+      .actions {
+        margin-left: $toolbar-actions-margin-left;
+      }
+      .actions ::ng-deep * {
```

Comment:
Spacing above. 

And styling-wise for other components since we uses css it would probably make more sense to do:

```scss
.actions {
  ::ng-deep {
    * {
      color: $toolbar-text-color;
    }
  }
}
```